tslint.json

```diff
@@ -11,7 +11,10 @@
   "rules": {
     "semicolon": false,
     "interface-name": false,
-    "no-console": false,
+    "no-console": {
+      "severity": "warning",
+      "options": ["debug", "info", "log", "time", "timeEnd", "trace"]
```

Comment:
Since by default all methods are banned, is it worth to specify them manually? This leads to some omissions, like `.table`, `.assert`, `.count` and plenty of other methods are allowed for some reason.